Admin/AbstractAdmin.php

```diff
@@ -2729,61 +2729,61 @@ public function hasAccess($action, $object = null)
     /**
      * {@inheritdoc}
      */
-    public function configureActionButtons($action, $object = null)
+    public function configureActionButtons($actionList, $action, $object = null)
```

Comment:
`actionsList`

This naming is still weird to me.

Just reading the signature, \wWhy configureActionButton**s** get a action list and a single action?